server/odc-service/src/main/java/com/oceanbase/odc/service/session/initializer/SwitchSchemaInitializer.java

```diff
@@ -48,11 +50,13 @@ public void init(Connection connection) throws SQLException {
         if (this.connectionSession.isExpired()) {
             return;
         }
-        String schema = ConnectionSessionUtil.getCurrentSchema(this.connectionSession);
-        if (connection == null || schema == null) {
+        String currentSchema = ConnectionSessionUtil.getCurrentSchema(this.connectionSession);
```

Comment:
Can this logic work properly in the following scenario?
1. I created a session with connectSchema A;
2. I switched it into currentSchema B;
3. The session was expired, then reconnected.
I wonder how to know which currentSchema is in this case.

Reply:
odc does not support switch schema currently.

Unfortunately, we can not switch to the schema that he/she switch to before if we supports schema in the future and the connect session is reconnected